Fixing I001 in file with Cr Lf line endings fails after 100 iterations

With `ruff 0.0.203` the reproducer from #1477 now results in a loop, and the `I001` linting error ends up not being fixed.

```
$ echo -e -n '"""A file with CRLF line endings."""\r\nimport sys\r\nimport os\r\n' > crlf.py

$ python -c "from pathlib import Path;print(Path('crlf.py').read_bytes())"
b'"""A file with CRLF line endings."""\r\nimport sys\r\nimport os\r\n'

$ ruff crlf.py --select I001 --fix

warning: Failed to converge after 100 iterations.

This likely indicates a bug in `ruff`. If you could open an issue at:

https://github.com/charliermarsh/ruff/issues

quoting the contents of `crlf.py`, along with the `pyproject.toml` settings and executed command, we'd be very appreciative!

crlf.py:2:1: I001 Import block is un-sorted or un-formatted
Found 101 error(s) (100 fixed, 1 remaining).
```

--- a/src/isort/plugins.rs
+++ b/src/isort/plugins.rs
@@ -95,7 +95,7 @@ pub fn check_imports(
         end_location: Location::new(range.end_location.row() + 1 + num_trailing_lines, 0),
     };
     let actual = dedent(&locator.slice_source_code_range(&range));
-    if actual == expected {
+    if actual == dedent(&expected) {
         None
     } else {
         let mut check = Check::new(CheckKind::UnsortedImports, range);

